js-array-methods-2/js-array-methods-2_animal-objects/index.js

```diff
@@ -91,22 +95,46 @@ const indexOfAnimalWithNameLongerFive = null;
 // Hint: sort() mutates the original array, which is bad.
 // -> Use animals.slice().sort(...) to make a copy (and the tests work).
 
-const animalsSortedAlphabetically = null;
+// in the example it's enough to do the following (since no capital letters etc. and no special signs):
+const animalsSortedAlphabetically = animals
+  .map((animal) => animal.name)
+  .slice()
+  .sort();
```

Comment:
Thats my solution: 

const animalsSortedAlphabetically = animals.slice().sort((a, b) => {
  if (a.name < b.name) {
    return -1;
  }
  if (a.name > b.name) {
    return 1;
  }
  return 0;
});

But this one looks good too and if it works even better :)